dev/io.openliberty.microprofile.openapi.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -23,4 +23,16 @@ docPath=MicroProfile OpenAPI document endpoint path
 docPath.desc=Specifies the URL path for obtaining OpenAPI documents. The URL path must be constructed with Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.).
 
 uiPath=MicroProfile OpenAPI UI endpoint path
-uiPath.desc=Specifies the URL path for accessing the MicroProfile OpenAPI UI. The URL path must be constructed with Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.) If no value is specified, the path is set to $docPath/ui.
\ No newline at end of file
+uiPath.desc=Specifies the URL path for accessing the MicroProfile OpenAPI UI. The URL path must be constructed with Unicode alphanumeric characters A-Za-z0-9, underscore (_), dash (-), forward slash (/), or period (.) If no value is specified, the path is set to $docPath/ui.
+
+includeApplication=MicroProfile OpenAPI included applications
+includeApplication.desc=Application names, one per element, that are to be included in the OpenAPI document. The name of an application may be defined in the server.xml, otherwise it defaults to the application archive filename excluding any extension.  The "all" special value includes all available applications. 
```

Comment:
replace the word "may" with the word "can", i.e.,
The name of an application can be defined

"May" is avoided because it can confuse translators, as it can mean possibility or granting of permission.